src/Data/String.agda

```diff
@@ -22,3 +22,9 @@ open import Data.String.Properties using (_≈?_; _≟_; _<?_; _==_) public
 
 toVec : (s : String) → Vec Char (length s)
 toVec s = Vec.fromList (toList s)
+
```

Comment:
Can we have a more general `between : String → String → String → String` too?

Reply:
Also, this should probably go in `Base`.

Reply:
Is `between` really a useful specialisation of `xs ++ ys ++ zs` though? I'm not 100% sold on the idea.